### DISCUSSION TITLE
Integrate Bluetooth audio transmission

### DISCUSSION THREAD

**Discussion by olderzeus** [darthcloud/BlueRetro#1051] [Ideas]
ESP32 has Bluetooth audio function. If possible, it can also transmit game console audio to Bluetooth earphones through BlueRetro devices.

**Comment by patrikmckane** [comment]
You can use something like bluetoth transmiter from aliexpress. for blueretro adapter this function is totaly not necesary and totaly miss the point of this adapter.

**Reply by olderzeus** [reply]
Thanks for your reply

**Reply by rutdarkronin** [reply]
I highly disagree with above statement. It's always nice to have options for those who wants them. DS4, DualSense, XOne controllers can output audio into 3.5 jack so why not use it for those who keen on HW2 internal solutions? 
Overall it's a pretty nice feature to have. And Ali BT adapters cannot be paired with most of headphones in a normal way.

**Reply by patrikmckane** [reply]
> I highly disagree with above statement. It's always nice to have options for those who wants them. DS4, DualSense, XOne controllers can output audio into 3.5 jack so why not use it for those who keen on HW2 internal solutions? Overall it's a pretty nice feature to have. And Ali BT adapters cannot be paired with most of headphones in a normal way.

First of All bluetetro is nôt capable of streaming audio to controller like DS4 or DS5. So yeah i stand for what i wrote.
